src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -21,96 +21,77 @@
 
 <template>
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
-		<div class="new-conversation"
-			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
-				class="conversations-search"
-				:is-searching="isSearching"
-				@input="debounceFetchSearchResults"
-				@submit="onInputEnter"
-				@keydown.enter.native="handleEnter"
+		<div class="new-conversation" :class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
+			<SearchBox v-model="searchText" class="conversations-search" :is-searching="isSearching"
+				@input="debounceFetchSearchResults" @submit="onInputEnter" @keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
 			<NewGroupConversation v-if="canStartConversations" />
 		</div>
+		<div class="button-container">
+			<button class="button" @click="insertValue('is:unread')" :title="t('spreed', 'Unread messages')">{{
```

Comment:
We need to use `NcButton` from Vue components library. It comes in spreed repo as node_modules dependencies, so you could take a look, how it is used in other components; there should be a plenty of examples

Reply:
I was using the NcButton BUT it wasn't flexible with the modification( changing the span font size ) and I switched to the normal Button tag with which I could modify the style adequately.

Reply:
What you're intending to do, should be possible by using `:deep()` selector from Vue.
Try to experiment with it to understand how it works, but as I said before, we should stick to the native appearance of NcButton, because it was made bold for a reason
And also I spend a week or two replacing buttons in project with NcButton, so don't ruin my job)